### ISSUE TITLE
Unresolved var false positive using foo/bar.baz style reference in .cljc files

### ISSUE THREAD

**Issue by anderseknert** [clj-kondo/clj-kondo#1818]
**version**

clj-kondo v2022.09.08

**platform**

Native, Mac OS 

**problem**

Using the "foo/bar.baz" style of reference sometimes leads to "unresolved var" being reported incorrectly. 

**repro**

```clojure
(ns main
  #?(:cljs (:require [cljs.nodejs :as node])))

(defn -main []
  #?(:clj  (println "clj")
     :cljs (println node/process.env)))
```

```
❯ clj-kondo --lint main.cljc
main.cljc:6:22: warning: Unresolved var: node/process.env
```
**config**

Same behavioir with / without config

**expected behavior**

No linter error.

**Comment by anderseknert** [comment]
Thanks @borkdude 😊
